src/components/UserProperties.vue

```diff
@@ -3,39 +3,36 @@
 </template>
 
 <script>
-import axios from "axios";
+import UserService from '@/services/UserService.js'
+import { mapState, mapActions } from 'vuex'
 
 export default {
+  computed: mapState(['isLoggedIn', 'userId', 'configs']),
   methods: {
+    // object spread operator
+    // https://vuex.vuejs.org/guide/state.html#object-spread-operator
+    ...mapActions(['saveConfigs']),
+
     saveLocalUserConfigs() {
```

Comment:
Should be in `UserService`?

Reply:
This can't be moved to UserService.js because here, a Vuex action is being dispatched which cannot be done from a JS file. It can only be done from a component

Reply:
Hmm... what is the action that is being dispatched?

1. So then having the `UserService` does't really make sense. If it's not containing all the logic, and the `User` logic is spread across multiple files, no point
OR
2. Design it so that the vuex action is separated from the logic
OR
3. Make `UserService` a component? 

Note that I don't really know which one of the above is possible/feasible/etc. My main point here is that if you're creating a `UserService` class/file, then that should be responsible for `User` stuff, no?

Reply:
https://stackoverflow.com/questions/46716709/how-to-dispatch-action-from-a-javascript-file-instead-of-a-vue-component

Maybe this will work? 

OR this: https://forum.vuejs.org/t/how-to-access-vuex-store-from-a-js-module/10124 

?

Reply:
I will try these out
The point of having the `UserService` file was to handle and encapsulate the api calls
I agree that this logic should not be dangling around.
Ideally the user config should be fetched by store and saved there itself
and any component can access the stored config by accessing Vuex


I will try these out, until then,I'll merge this PR as Vaibhav has to put out a release